samples/grid-lite/demo/view-chart-data-using-grid/demo.html

```diff
@@ -0,0 +1,15 @@
+<script src="https://cdn.jsdelivr.net/npm/@highcharts/grid-lite/grid-lite.js"></script>
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/modules/exporting.js"></script>
+<script src="https://code.highcharts.com/modules/export-data.js"></script>
+
+<div class="content">
+    <div id="container">
+        <div id="chart"></div>
+    </div>
+    <p class="highcharts-description">
+        This demo uses Grid to show the data for a chart instead of the one built into the Export module.<br><br>
+        Use the context menu and click "View data table" to show the data using Grid.<br><br>
+        Using Grid to display the chart data will give you all the functionality Grid lite has to offer and a nice looking table out of the box.
+    </p>
```

Comment:
```suggestion
    <p class="highcharts-description">
    This demo uses Grid to show the data for a chart instead of the default data table built into the Export module. Using Grid to display the chart data will give you all the functionality Grid lite has to offer and a nice looking table out of the box.<br><br>
        Click the top right menu icon and then "View data table" to show the data using Grid.
    </p>
```

Just reshuffled content, and removed "context menu". IMO this is per definition not a context menu.